### PR TITLE
Migrate to Guava 20.0.

### PR DESCRIPTION
**Pull request by armiol**
Summary of changes:

* migrated to Guava 20.0;
* code usages have been updated;
* Spine version was bumped to `0.6.4-SNAPSHOT`.

### CODE REVIEW

**Comment by armiol** [comment]
@dmdashenkov PTAL.

File: server/src/test/java/org/spine3/server/storage/AbstractStorageShould.java [hunk 1]
```diff
@@ -73,15 +72,13 @@ public void tearDownAbstractStorageTest() {
 
     /**
      * Closes the storage and propagates an exception if any occurs.
-     *
-     * @see Throwables#propagate(Throwable)
```
Comment (dmdashenkov) [inline server/src/test/java/org/spine3/server/storage/AbstractStorageShould.java]: Maybe we should keep `@see` block with updated API usage?

Comment (armiol) [inline server/src/test/java/org/spine3/server/storage/AbstractStorageShould.java]: `Throwables#propagate` will be removed in the 21.0.

That's why we don't mention it anymore.

**Comment by armiol** [comment]
@dmdashenkov updated a single usage of `new RuntimeException(..)` with `Exceptions.wrapped(..)`.

Please see my answer inline as well.

PTAL.

**Pull request merged**